pkg/storage/v2/mysql/client.go

```diff
@@ -179,6 +186,7 @@ func (c *client) QueryRowContext(ctx context.Context, query string, args ...inte
 	return r
 }
 
+// Deprecated
 func (c *client) BeginTx(ctx context.Context) (Transaction, error) {
```

Comment:
Is `Deprecated` here mean that this function will not be exported anymore and internal use only?

Reply:
@hvn2k1 If transaction processing is required, we plan to unify the use of RunInTransactionV2 and eventually make BeginTx private. Any concerns or ideas?

Reply:
No I just wondered its meaning here, it's all clear now, thank you!